docs/CONTRIBUTING.md

```diff
@@ -171,8 +171,24 @@ docker exec -it student_dashboard_mysql mysql_upgrade \
 When prompted, specify the password for the root MySQL user.
 It should be found in the `MYSQL.ROOT_PASSWORD` property of `env.hjson`.
 
-[Next: Contributors](../docs/CONTRIBUTORS.md)
+### Backing up the development database
+
+To backup the development databaes with the default settings, use this command
+```sh
+docker exec student_dashboard_mysql mysqldump --socket=/tmp/mysql.sock -uroot -pstudent_dashboard_root_pw student_dashboard > student_dashboard.sql
```

Comment:
The default mysql_user id in .env.sample file is `student_dashboard_user` with password `student_dashboard_pw`. Should we change the user name to `student_dashboard_user` instead of `root`?

Reply:
The regular student_dashboard_user didn't have the correct permissions to run a mysqldump. It needs to run as root user here. We could possibly give it that user the permissions it needs.